File: 0275-h-index-ii/0275-h-index-ii.rs
impl Solution {
    pub fn h_index(citations: Vec<i32>) -> i32 {
        let mut ans = citations.len() as i32;
        for i in 0..citations.len() {
            if citations[i] >= ans {
                return ans;
            }
            ans -= 1
        }
        ans
    }
}

File: leetcode/0275-h-index-ii/0275-h-index-ii.cpp
class Solution {
public:
    int hIndex(vector<int>& citations) {
        int ans = citations.size();
        for (int i = 0; i < citations.size(); i++) {
            if (citations[i] >= ans) return ans;
            ans--;
        }
        return ans; 
    }
};